website/content/docs/platform/k8s/csi/installation.mdx

```diff
@@ -49,3 +50,80 @@ You can see all the available values settings by running `helm inspect values ha
 Docs](/docs/platform/k8s/helm/configuration). Commonly used values in the Helm
 chart include limiting the namespaces the Vault CSI Provider runs in, TLS options and
 more.
+
+## Installation on OpenShift
+
+We recommend using the [Vault agent injector on Openshift](/docs/platform/k8s/helm/openshift)
+instead of the Secrets Store CSI driver. OpenShift
+[does not recommend](https://docs.openshift.com/container-platform/4.9/storage/persistent_storage/persistent-storage-hostpath.html)
```

Comment:
I wasn't sure if this was explained elsewhere about why it's not recommended to use 'hosPath' mounting in production or helm charts using CSI objects.  Does it warrant a need for further explanation to alert the users not to use it in a production environment? I clicked the link but I didn't see a thorough explanation or I might've missed it. Maybe it's not needed, but I wanted to mention it anyway.

Reply:
It does not seem to be in the documentation directly, I see a few indirect references. Adding some information to clarify that privileged pods will provide elevated access not recommended for production.